node/middlewares/e2e.ts

```diff
@@ -0,0 +1,43 @@
+import parse from 'co-body'
+
+export async function onSpecComplete(
+  ctx: Context,
+  next: () => Promise<unknown>
+) {
+  const {
+    clients: { healthcheck },
+    headers,
+    req,
+  } = ctx
+
+  const specReport = (await parse.json(req)) as SpecReport
+
+  // example duration: 01m42s
+  const duration = `${new Date(specReport.report?.stats.duration ?? 0)
+    .toISOString()
+    .slice(14, -5)
+    .replace(':', 'm')}s`
```

Comment:
nit: do you think it's better to use `Intl.DateTimeFormat` for this?

Reply:
Hmm, I'm not sure how I would accomplish the same thing with this. Do you?

Tbh, I'm starting to think that it would be better to just straight up convert the `specReport.report?.stats.duration` using math, because it seems odd to me to use a date to represent a duration. Makes sense?
```suggestion
  const durationInSeconds = Math.floor((specReport.report?.stats.duration ?? 0) / 1000)
  const formattedDuration = `${Math.floor(durationInSeconds / 60)}m${durationInSeconds % 60}s`

```

Reply:
yeah, seems better to me! I thought that `duration` was an ISO datetime string, that's why I brought up date time format